client/src/elements/schema-editor/schema-editor-files.js

```diff
@@ -56,8 +56,19 @@ export class SchemaEditorFiles {
 
     const QServerBaseUrl = await qEnv.QServerBaseUrl;
 
+    let dictDefaultMessageTranslation = this.i18n.tr(
+      "dropzone.dictDefaultMessage"
+    );
+    if (this.options.maxFiles) {
```

Comment:
Imho you do not need this extra check here, if `this.options.maxFiles` is undefined it's not equal 1 anyways. Just if it's possible that `this.options` is undefined you should check that first, i.e. `if (this.options && this.options.maxFiles === 1)`

Reply:
Fixed in https://github.com/nzzdev/Q-editor/pull/90/commits/f57c04c9e35bffc53f6f4310549208b3d9a31a19. This branch is now delpoyed on staging